pulsar-broker/src/test/java/org/apache/pulsar/broker/service/ReplicatorTopicPoliciesTest.java

```diff
@@ -296,6 +298,37 @@ public void testReplicatorMessageDispatchRatePolicies() throws Exception {
                 assertNull(admin3.topicPolicies(true).getDispatchRate(persistentTopicName)));
     }
 
+    @Test
+    public void testReplicatorOffloadPolicies() throws Exception {
+        final String namespace = "pulsar/partitionedNs-" + UUID.randomUUID();
+        final String persistentTopicName = "persistent://" + namespace + "/topic" + UUID.randomUUID();
+
+        init(namespace, persistentTopicName);
+        OffloadPoliciesImpl offloadPolicies =
+                OffloadPoliciesImpl.create("s3", "region", "bucket", "endpoint", null, null, null, null,
+                8, 9, 10L, null, OffloadedReadPriority.BOOKKEEPER_FIRST);
+
+        // set offload policies
+        try{
+            admin1.topicPolicies(true).setOffloadPolicies(persistentTopicName, offloadPolicies);
+        }catch (Exception ignore){
+            // ignore PersistentTopicsBase#internalUpdateOffloadPolicies not found nar loader exception.
```

Comment:
We cannot ignore this.
In a test case like this you have 100% control over what's happening.

Please drop this catch.

Reply:
If i do not ignore this exception, i will get  not found nar loader exception. 

because of this test is not mock test, should i add a nar package in project ?